api/db/migrate/20181019100654_add_preference_to_player.rb

```diff
@@ -0,0 +1,5 @@
+class AddPreferenceToPlayer < ActiveRecord::Migration[5.2]
+  def change
+    add_column :players, :preference, :int, default: 0
```

Comment:
How about saving it as a string column? I personally find it easier to add new values, as well as write future migrations. What's your opinion on that?

Reply:
No preference

Reply:
Then let's go with string keys, easier to reason about and there are just a few :)